apps/condo/domains/meter/access/MeterReading.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * Generated by `createschema meter.MeterReading 'organization:Relationship:Organization:CASCADE; property:Relationship:Property:CASCADE; account?:Relationship:BillingAccount:SET_NULL; billingAccountMeter?:Relationship:BillingAccountMeter:SET_NULL; date:DateTimeUtc; meter:Relationship:Meter:CASCADE; value:Integer; source:Relationship:MeterReadingSource:PROTECT'`
+ */
+const { throwAuthenticationError } = require('@condo/domains/common/utils/apolloErrorFormatter')
+const { Property } = require('@condo/domains/property/utils/serverSchema')
+const { checkPermissionInUserOrganizationOrRelatedOrganization } = require('../../organization/utils/accessSchema')
+const { RESIDENT } = require('@condo/domains/user/constants/common')
+const { queryOrganizationEmployeeFromRelatedOrganizationFor, queryOrganizationEmployeeFor } = require('@condo/domains/organization/utils/accessSchema')
+const { Resident: ResidentServerUtils } = require('@condo/domains/resident/utils/serverSchema')
+const { get, uniq, compact } = require('lodash')
+
+
+async function canReadMeterReadings ({ authentication: { item: user }, context }) {
+    if (!user) return throwAuthenticationError()
+    if (user.isAdmin) return {}
+
+    const userId = user.id
+    if (user.type === RESIDENT) {
+        const residents = await ResidentServerUtils.getAll(context, { user: { id: userId } })
+        if (residents.length === 0) {
+            return false
+        }
+        const organizations = compact(residents.map(resident => get(resident, ['organization', 'id'])))
+        if (organizations.length > 0) {
+            return {
+                organization: { id_in: uniq(organizations) },
```

Comment:
it look like a security bug?! need a test

Reply:
fix and write tests for it in f9731f3 and 3f80308